README.md

```diff
@@ -201,6 +201,42 @@ var railsImagePath = "<%= helpers.image_path('rails.png') %>";
 
 This is enabled by the `rails-erb-loader` loader rule in `config/webpack/shared.js`.
 
+## Ready for Elm
+
+To use Webpacker with [Elm](http://elm-lang.org), create a new app with `rails new myapp
+--webpack=elm` (or run `rails webpacker:install:elm` on a Rails app already setup with webpacker).
+The Elm library and core packages will be added via Yarn and Elm itself. An example `Main.elm` app
+is also added to your project in `app/javascript` so that you can experiment with Elm right away.
+For example, you might want to make the following changes to your newly generated, Elm-enabled,
+Rails app:
+
+### Routes (routes.rb)
+
+    Rails.application.routes.draw do
+      resource :demo, controller: :demo, only: :show
+    end
+
+### Controllers
+
+    class DemoController < ApplicationController
+      def show
+      end
+    end
```

Comment:
Lets remove routes and controllers part to make things simple to follow up and just follow what other integration suggests (we are not using views anyway).